signing_service/signing_service/signing_service.py

```diff
@@ -102,6 +112,9 @@ def __init__(
         self.was_sigterm_caught: bool = False
         self.reconnection_counter = 0
         self.maximum_reconnection_attempts = maximum_reconnect_attempts
+        self.notifier = notifier
+        self.signing_service_daily_transaction_sum_so_far = self._get_signing_service_daily_transaction_sum_so_far()
+        self.last_signed_transaction = datetime.datetime.now().strftime('%Y-%m-%d')
```

Comment:
This name is misleading, it is suggesting that you will have a `ethereum.Tranaction` object here, not a date.

Reply:
Done